packages/osd-monaco/src/ppl/language.ts

```diff
@@ -266,6 +357,45 @@ const mapPPLTokenToMonacoTokenType = (tokenType: string): string => {
     return 'predefined';
   }
 
+  // Relevance Function Parameters - Parameters for search functions
+  if (
+    [
+      'ALLOW_LEADING_WILDCARD',
+      'ANALYZE_WILDCARD',
+      'ANALYZER',
+      'AUTO_GENERATE_SYNONYMS_PHRASE_QUERY',
+      'BOOST',
+      'CUTOFF_FREQUENCY',
+      'DEFAULT_FIELD',
+      'DEFAULT_OPERATOR',
+      'ENABLE_POSITION_INCREMENTS',
+      'ESCAPE',
+      'FLAGS',
+      'FUZZY_MAX_EXPANSIONS',
+      'FUZZY_PREFIX_LENGTH',
+      'FUZZY_TRANSPOSITIONS',
+      'FUZZY_REWRITE',
+      'FUZZINESS',
+      'LENIENT',
+      'LOW_FREQ_OPERATOR',
+      'MAX_DETERMINIZED_STATES',
+      'MAX_EXPANSIONS',
+      'MINIMUM_SHOULD_MATCH',
+      'OPERATOR',
+      'PHRASE_SLOP',
+      'PREFIX_LENGTH',
+      'QUOTE_ANALYZER',
+      'QUOTE_FIELD_SUFFIX',
+      'REWRITE',
+      'SLOP',
+      'TIE_BREAKER',
+      'TYPE',
+      'ZERO_TERMS_QUERY',
+    ].includes(type)
+  ) {
+    return 'attribute';
+  }
+
```

Comment:
not a blocker for your PR again, but why are these PPL keywords just inline defined in this file? What do we do if we want to expand the PPL language? we gotta find in this code file?

This is not good